lazer/contracts/sui/sdk/js/src/client.ts

```diff
@@ -0,0 +1,28 @@
+import { bcs } from "@mysten/sui/bcs";
+import { Transaction } from "@mysten/sui/transactions";
+import { SUI_CLOCK_OBJECT_ID } from "@mysten/sui/utils";
+
+const MAX_ARGUMENT_SIZE = 16 * 1024;
+
+export function addParseAndVerifyLeEcdsaUpdateCall(opts: {
+  tx: Transaction;
+  packageId: string;
+  stateObjectId: string;
+  updateBytes: Buffer;
+}) {
+  const { tx, packageId, stateObjectId, updateBytes } = opts;
```

Comment:
Super minor but in case you aren't aware, you can destructure as part of the function signature e.g.

```ts
export const addParseAndVerifyLeEcdsaUpdateCall = (
  { tx, packageId, stateObjectId, updateBytes }: {
    tx: Transaction;
    packageId: string;
    stateObjectId: string;
    updateBytes: Buffer;
  }
) => {
  // ...
}